README.md

```diff
@@ -51,45 +51,36 @@ Install it
 
     bundle install
 
-### Rails 4.x.x
+Add to your `app/assets/javascripts/application.js` file
 
-Run the generator
-
-    rails generate autocomplete:install
-
-And include autocomplete-rails.js on your layouts
-
-    javascript_include_tag "autocomplete-rails.js"
-
-#### Upgrading from older versions
+    //= require jquery
+    //= require jquery_ujs
+    //= require jquery-ui
+    //= require autocomplete-rails
 
-If you are upgrading from a previous version, run the generator after installing to replace the javascript file.
+## Usage
 
-    rails generate autocomplete:install
+### Demo Application by [Yifei](https://github.com/yifeiwu)
 
-I'd recommend you do this every time you update to make sure you have the latest JS file.
+A live demo can be seen 
 
-#### Uncompressed Javascript file
+[here](https://rocky-thicket-9286.herokuapp.com/)
 
-If you want to make changes to the JS file, you can install the
-uncompressed version by running:
+[source](https://github.com/yifeiwu/rails4-autocomplete-demo)
 
-    rails generate autocomplete:uncompressed
+As a new developer, I had some issues getting this to work by following the documentation. However after trying some things and reading [Yoni Weisbrod](http://www.yoniweisbrod.com/autocomplete-magic-with-rails/)'s blog post, I was able to make the autocomplete work and implement a few useful features. 
 
-### Rails 4 and higher
+##Some implemented features
 
-Just add it to your app/assets/javascripts/application.js file
+1. The css has been changed such that the results show up better against the box of suggestions. See  `<app/assets/stylesheets/food.scss>` for details. I obtained this from a gist(forgive me I don't remember who the author is at the moment, please contact me if you do and I'll give credit). Upon mouseover/arrowkey presses, the selection will be highlighted. 
 
-    //= require jquery
-    //= require jquery_ujs
-    //= require jquery-ui
-    //= require autocomplete-rails
+2. One Yoni's improvements, you can click on a suggested item to submit the search, instead of having to click on the submit button after clicking on the item. This is an example of how to hook onto the *railsAutocomplete.select* event. 
 
-## Usage
+3. The autocomplete is implemented in the context of a search form with a simple scope search(see the food model and controller). 
 
 ### Model Example
 
-Assuming you have a Brand model:
+Assuming you have a Brand model that contains a name attribute:
 
     class Brand < ActiveRecord::Base
```

Comment:
Rather than the class, I suggest entering a sample schema table, that shows the attributes and their data types.